README.md

```diff
@@ -42,3 +42,12 @@ drush sqlq "SHOW TABLES LIKE '_raw_%';" | awk -v ORS=, '{ print $1 }' | sed 's/,
 # Replace teh drush alias with your own.
 `drush @remote-alias sql-connect` < raws.sql
 ```
+
+# Scan a directory.
+
+There's an option to scan a whole directory instead of giving an exact path to a ``csv`` file.
+Execute the command: ``drush csv2sql --dir=/PATH/TO/directory``
+
+Also there's an option to scan a directory and add a direct path to a file together.
+Execute the command: ``drush csv2sql /PATH/TO/file.csv --dir=/PATH/TO/directory``
```

Comment:
@amitaibu Scanning a directory and wanting to add another file from another directory. We can remove the documentation if you want, But it won't effect the flow of the code if we keep it or leave it.